app/views/admin/loans/questionnaires/_questionnaire_group.html.slim

```diff
@@ -57,62 +57,70 @@
 
-            - if value.has_breakeven_table?
-              - if value.breakeven_report.present?
+            - if response.has_breakeven_table?
+              - if response.breakeven_report.present?
                 .view-element
-                  = render "admin/loans/questionnaires/breakeven_table_report", report: value.breakeven_report
-                  = render "admin/loans/questionnaires/breakeven_charts", report: value.breakeven_report, id: value.loan_question.id
-                  - if value.breakeven_report[:periods] > 1
-                    = render "admin/loans/questionnaires/breakeven_rampup/main", report: value.breakeven_report
+                  = render "admin/loans/questionnaires/breakeven_table_report", report: response.breakeven_report
+                  = render "admin/loans/questionnaires/breakeven_charts", report: response.breakeven_report, id: response.loan_question.id
+                  - if response.breakeven_report[:periods] > 1
+                    = render "admin/loans/questionnaires/breakeven_rampup/main", report: response.breakeven_report
                 .form-element
                   = render 'admin/loans/questionnaires/breakeven_table_question', f: f,
-                    question: question, data: value.breakeven_hash, total_fixed_costs: value.breakeven_report[:total_fixed_costs]
+                    question: question, data: response.breakeven_hash, total_fixed_costs: response.breakeven_report[:total_fixed_costs]
               - else
                 .form-element
                   = render 'admin/loans/questionnaires/breakeven_table_question', f: f,
-                    question: question, data: value.breakeven_hash, total_fixed_costs: 0
+                    question: question, data: response.breakeven_hash, total_fixed_costs: 0
 
-            - if value.has_business_canvas?
-              .answer class=('form-element' if value.business_canvas_blank?)
+            - if response.has_business_canvas?
+              .answer class=('form-element' if response.business_canvas_blank?)
                 = render 'admin/loans/questionnaires/business_model_canvas', f: f,
-                  question: question, data: value.business_canvas
+                  question: question, data: response.business_canvas
 
-            - if value.has_linked_document?
+            - if response.has_linked_document?
               = render 'admin/loans/questionnaires/linked_document', f: f,
-                question: question, document: value.linked_document
+                question: question, document: response.linked_document
 
-            - if value.blank?
-              .view-element.answer.blank = t("loan.no_answer")
+            .not-applicable
+              - if response.not_applicable?
+                .view-element.answer = t("loan.not_applicable")
+
+              .form-element
+                /= f.input_field "#{question.attribute_sym}[not_applicable]", as: :boolean
```

Comment:
I can use help here. I'd like to use Simple Form for this part, but am getting an error. Any ideas welcome. Thank you!